### DISCUSSION TITLE
PHP Fatal error:  Interface 'JsonSerializable' not found

### DISCUSSION THREAD

**Discussion by davidmarin77** [laravel/framework#33344] [General] [answered]
- Laravel Version: 7.
- PHP Version: 7.2
- Database Driver & Version:

Description:
when i install the laravel project in hostgator i got this issue

/vendor/laravel/framework/src/Illuminate/Support/Enumerable.php on line 11

Steps To Reproduce:

interface Enumerable extends Arrayable, Countable, IteratorAggregate, Jsonable, **JsonSerializable**

**Comment by GrahamCampbell** [comment, marked as answer]
This happens if `ext-json` is not enabled. Please ask your hosting provider to enable it. Laravel requires that extension.

**Reply by GrahamCampbell** [reply]
Indeed, composer will have forced you to have it: https://github.com/laravel/framework/blob/7.x/composer.json#L19.
